.dependabot/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    default_reviewers:
+      - "karelhala"
```

Comment:
I don't think karelhala would like to review the PRs of this project :D

Reply:
OH! awesome thank you. lol